tests/e2e_dbt_project/external_seeders/spark.py

```
@@ -0,0 +1,88 @@
+                if not cols:
+                    print(f"  Skipping {table_name} (completely empty file)")
+                    continue
+                col_defs = ", ".join(f"{q(c)} STRING" for c in cols)
+                sql = (
+                    f"CREATE TABLE IF NOT EXISTS "
+                    f"{q(seed_schema)}.{q(table_name)} ({col_defs}) USING delta"
+                )
+                print(f"  Creating empty table: {table_name}")
+                try:
+                    cursor.execute(sql)
+                except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)